fetch_napari_data.py

```diff
@@ -77,99 +159,70 @@ def build_plugins_dataframe():
     df = pd.DataFrame(all_plugin_data)
     return df
 
-# Define the function to extract the author's name from the email field
-def extract_author_name(email):
-
-    if not isinstance(email, str):
-        return ''
-
-    # Split the string by comma to process multiple authors
-    authors = email.split(',')
-    clean_authors = []
-
-    for author in authors:
-        # Use regex to find name patterns before the email
-        match = re.match(r'(.*)<.*?>', author)
-
-        if match:
-            # Remove surrounding quotation marks using strip
-            clean_author = match.group(1).replace('"', '').strip()
-            clean_authors.append(clean_author)
-        else:
-            clean_authors.append(author.replace('"', '').strip())
-
-    # Return the list of clean author names
-    return ', '.join(clean_authors)
-
-# Function to classify the 'home' column
-def classify_website(home_url):
-    if pd.notnull(home_url):
-        if 'pypi.org' in home_url:
-            return 'pypi'
-        elif 'github.com' in home_url:
-            return 'github'
-    return 'other'
-
 
 if __name__ == "__main__":
+    # Create target build directory and a data directory
     build_dir = sys.argv[1] if len(sys.argv) > 1 else '.'
     data_dir = f'{build_dir}/data'
 
+    # Create and populate a raw DataFrame with plugin data
     df_plugins = build_plugins_dataframe()
 
-    # Drop columns where all elements are NaN
-    #df_plugins.dropna(axis=1, how='all', inplace=True)
-
-    # Drop columns where less than 20 non-NaN counts
+    # Clean raw DataFrame by removing columns that are mostly empty, keep columns that have at least 20 non-missing values
     df_plugins.dropna(axis=1, thresh=20, inplace=True)
 
-    # Create a dictionary of column names and their non-NaN counts
+    # Create a dictionary of column names and their count of values and print sorted dictionary
     column_counts = {column: df_plugins[column].count() for column in df_plugins.columns}
-
-    # Sort the dictionary by counts and print
     for column, count in sorted(column_counts.items(), key=lambda item: item[1],  reverse=True):
         print(column, count)
```

Comment:
While we're here let's just remove this print, it's noisy and not particularly useful

Reply:
@DragaDoncila I removed the print and created a sorted dictionary. It wasn't clear to me if this was used in anything other than the print.